lab-raziyeh/model/client.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const uuid = require('node-uuid');
+
+const Client = module.exports = function(socket){
+  this.socket = socket;
+  this.nickname = `user_${Math.random()}`;
+  this.id = uuid.v4();
+};
```

Comment:
@Raziyehbazargan this character signifies no new line was created at the end of the file - if you add an additional line break after your last line of code, prior to pushing to GitHub, this should remove this problem